pkg/cache/cache.go

```diff
@@ -140,6 +141,7 @@ type ClusterQueue struct {
 	NamespaceSelector labels.Selector
 	Preemption        kueue.ClusterQueuePreemption
 	Status            metrics.ClusterQueueStatus
+	FlavorFungibility v1beta1.FlavorFungibility
```

Comment:
Where is this populated?

Reply:
I removed this field, we can get this from cq.Preemption